<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BreakdownDialog</class>
 <widget class="QDialog" name="BreakdownDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>424</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Unknown Word</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <widget class="QLabel" name="wordLabel">
     <property name="text">
      <string>Break down the word: BIGPLACEHOLDERWORD</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QPushButton" name="aaBut">
       <property name="text">
        <string>AA</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="awBut">
       <property name="text">
        <string>AW</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="erBut">
       <property name="text">
        <string>ER</string>
       </property>
      </widget>
     </item>
     <item row="12" column="2">
      <widget class="QPushButton" name="eBut">
       <property name="text">
        <string>E</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="chBut">
       <property name="text">
        <string>CH</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="ahBut">
       <property name="text">
        <string>AH</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="iyBut">
       <property name="text">
        <string>IY</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QPushButton" name="uhBut">
       <property name="text">
        <string>UH</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="ehBut">
       <property name="text">
        <string>EH</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QPushButton" name="aiBut">
       <property name="text">
        <string>AI</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QPushButton" name="sBut">
       <property name="text">
        <string>S</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QPushButton" name="nBut">
       <property name="text">
        <string>N</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QPushButton" name="kBut">
       <property name="text">
        <string>K</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QPushButton" name="rBut">
       <property name="text">
        <string>R</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="jhBut">
       <property name="text">
        <string>JH</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QPushButton" name="owBut">
       <property name="text">
        <string>OW</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QPushButton" name="mBut">
       <property name="text">
        <string>M</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="dhBut">
       <property name="text">
        <string>DH</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="ayBut">
       <property name="text">
        <string>AY</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="lBut">
       <property name="text">
        <string>L</string>
       </property>
      </widget>
     </item>
     <item row="9" column="3">
      <widget class="QPushButton" name="vBut">
       <property name="text">
        <string>V</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QPushButton" name="oyBut">
       <property name="text">
        <string>OY</string>
       </property>
      </widget>
     </item>
     <item row="8" column="3">
      <widget class="QPushButton" name="tBut">
       <property name="text">
        <string>T</string>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QPushButton" name="shBut">
       <property name="text">
        <string>SH</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QPushButton" name="oBut">
       <property name="text">
        <string>O</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QPushButton" name="thBut">
       <property name="text">
        <string>TH</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="gBut">
       <property name="text">
        <string>G</string>
       </property>
      </widget>
     </item>
     <item row="9" column="2">
      <widget class="QPushButton" name="uwBut">
       <property name="text">
        <string>UW</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="aoBut">
       <property name="text">
        <string>AO</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="silBut">
       <property name="text">
        <string>&lt;SIL&gt;</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="hhBut">
       <property name="text">
        <string>HH</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QPushButton" name="ngBut">
       <property name="text">
        <string>NG</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="bBut">
       <property name="text">
        <string>B</string>
       </property>
      </widget>
     </item>
     <item row="7" column="3">
      <widget class="QPushButton" name="pBut">
       <property name="text">
        <string>P</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="fBut">
       <property name="text">
        <string>F</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="dBut">
       <property name="text">
        <string>D</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="ihBut">
       <property name="text">
        <string>IH</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="eyBut">
       <property name="text">
        <string>EY</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="aeBut">
       <property name="text">
        <string>AE</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QPushButton" name="wBut">
       <property name="text">
        <string>W</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QPushButton" name="yBut">
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
     </item>
     <item row="10" column="2">
      <widget class="QPushButton" name="zBut">
       <property name="text">
        <string>Z</string>
       </property>
      </widget>
     </item>
     <item row="10" column="3">
      <widget class="QPushButton" name="zhBut">
       <property name="text">
        <string>ZH</string>
       </property>
      </widget>
     </item>
     <item row="12" column="3">
      <widget class="QPushButton" name="uBut">
       <property name="text">
        <string>U</string>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QPushButton" name="wqBut">
       <property name="text">
        <string>WQ</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QPushButton" name="mbpBut">
       <property name="text">
        <string>MBP</string>
       </property>
      </widget>
     </item>
     <item row="14" column="2">
      <widget class="QPushButton" name="fvBut">
       <property name="text">
        <string>FV</string>
       </property>
      </widget>
     </item>
     <item row="14" column="3">
      <widget class="QPushButton" name="etcBut">
       <property name="text">
        <string>etc</string>
       </property>
      </widget>
     </item>
     <item row="15" column="0">
      <widget class="QPushButton" name="restBut">
       <property name="text">
        <string>rest</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="breakdownEdit"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BreakdownDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BreakdownDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
